src/components/Button/index.js

```diff
@@ -142,6 +145,7 @@ const defaultProps = {
     style: [],
     innerStyles: [],
     textStyles: [],
+    default: false,
```

Comment:
I don't think we need to specify props for the `default` style if it's only affecting the hover style. We can replace this with `shouldShowDefaultHoverEffect`.

Reply:
Tried to keep it consistent with *success* and *danger* styles. Each button may be only 1 of *success, danger, default*. Would you say `shouldShowDefaultHoverEffect` is better?

Reply:
The button has a default style applied, so adding new props `default` just for the hovered seems confusing.